android/src/main/java/com/oney/WebRTCModule/WebRTCView.java

```diff
@@ -237,102 +116,10 @@ protected void onDetachedFromWindow() {
-                changed = true;
-            }
-            if (this.frameWidth != videoWidth) {
-                this.frameWidth = videoWidth;
-                changed = true;
-            }
-        }
-        if (changed) {
-            // The onFrameResolutionChanged method call executes on the
-            // surfaceViewRenderer's render Thread.
-            post(requestSurfaceViewRendererLayoutRunnable);
-        }
-    }
-
-    /**
      * {@inheritDoc}
      */
     @Override
     protected void onLayout(boolean changed, int l, int t, int r, int b) {
-        int height = b - t;
-        int width = r - l;
-
-        if (height == 0 || width == 0) {
-            l = t = r = b = 0;
-        } else {
-            int frameHeight;
-            int frameRotation;
-            int frameWidth;
-            ScalingType scalingType;
-
-            synchronized (layoutSyncRoot) {
-                frameHeight = this.frameHeight;
-                frameRotation = this.frameRotation;
-                frameWidth = this.frameWidth;
-                scalingType = this.scalingType;
-            }
-
-            SurfaceViewRenderer surfaceViewRenderer = getSurfaceViewRenderer();
-
-            switch (scalingType) {
-            case SCALE_ASPECT_FILL:
-                // Fill this ViewGroup with surfaceViewRenderer and the latter
-                // will take care of filling itself with the video similarly to
-                // the cover value the CSS property object-fit.
-                r = width;
-                l = 0;
-                b = height;
-                t = 0;
-                break;
-            case SCALE_ASPECT_FIT:
-            default:
-                // Lay surfaceViewRenderer out inside this ViewGroup in accord
-                // with the contain value of the CSS property object-fit.
-                // SurfaceViewRenderer will fill itself with the video similarly
-                // to the cover or contain value of the CSS property object-fit
-                // (which will not matter, eventually).
-                if (frameHeight == 0 || frameWidth == 0) {
-                    l = t = r = b = 0;
-                } else {
-                    float frameAspectRatio
-                        = (frameRotation % 180 == 0)
-                            ? frameWidth / (float) frameHeight
-                            : frameHeight / (float) frameWidth;
-                    Point frameDisplaySize
-                        = RendererCommon.getDisplaySize(
-                                scalingType,
-                                frameAspectRatio,
-                                width, height);
-
-                    l = (width - frameDisplaySize.x) / 2;
-                    t = (height - frameDisplaySize.y) / 2;
-                    r = l + frameDisplaySize.x;
-                    b = t + frameDisplaySize.y;
-                }
-                break;
-            }
-        }
         surfaceViewRenderer.layout(l, t, r, b);
```

Comment:
The line seems wrong because these `l`, `t`, `r`, `b` are relative to the parent. So this `WebRTCView` is being told to lay itself out in `l`, `t`, `r`, `b` and we want `surfaceViewRenderer` to fill this `WebRTCView` so the arguments provided to `sufraceViewRenderer.layout` should be like in the deleted `SCALE_ASPECT_FILL` case.